standard.md

```diff
@@ -6,18 +6,17 @@
-**9b) Data Privacy & Security** |	Projects that collect data must identify the types of data collected and stored and demonstrate that the project ensures the privacy and security of this data and has taken steps to prevent adverse impacts resulting from it’s collection, storage and distribution.
-**9c) Inappropriate & Illegal Content** |	Projects that collect, store or distribute content must have policies identifying inappropriate and illegal content such as child sexual abuse materials and mechanisms for detecting and moderating inappropriate/illegal content.
-**9d) Protection from harassment** |	If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users.
+**6. Mechanism for extracting data** |	If this project has non personally identifiable information there must be a mechanism for extracting or importing non personally identifiable information (PII) data from the system in a non-proprietary format.
+**7. Adherence to privacy and applicable laws** |	The project must state that it complies with relevant privacy laws, and all applicable international and domestic laws. Projects that collect data must identify the types of data collected and stored and demonstrate that the project ensures the privacy and security of this data and has taken steps to prevent adverse impacts resulting from its collection, storage and distribution.
+**8. Adherence to standards & best practices** |	Projects must demonstrate adherence to standards, best practices and/or principles. i.e. the principles for digital development.
+**9. Freedom of expression** |	All projects must have strategies in place to anticipate, respond to and minimize adverse impacts on freedom of expression where governments are believed to be using the project’s product or services for illegitimate or political purposes.
+**10. Inappropriate & illegal content** |	Projects that collect, store or distribute content must have policies identifying inappropriate and illegal content such as child sexual abuse materials and mechanisms for detecting and moderating inappropriate/illegal content.
+**11. Protection from harassment** |	If the project facilitates interactions with or between users or contributors there must be a mechanism for users and contributors to protect themselves against grief, abuse, and harassment. The project must have a mechanism to address the safety and security of underage users.
+**12. Do no harm** |  All projects must demonstrate that they have a process to address accussations that the digital public good is doing harm or is otherwise having a an adverse affect on society. 
```

Comment:
Here again, I think "do no harm" is a principle that can be adhered to within a project. If a digital public good is used by a "bad actor" to do harm that is hard to stop. 

One idea that occurs to me is the creation of software licenses and creative commons rights reservation indicators that allows the creator of the digital public good to revoke the right to use. I would reserve this for the most extreme cases, which I cannot imagine but likely exist, at first.

Reply:
Yes, that's where the [Ethical Source](https://ethicalsource.dev/) comes into play. Rather than modifying the standard, this falls in the category of reviewing which licenses do we approve for DPGs. One such conversation is happening on unicef/publicgoods-candidates#115, so I'm resolving this conversation, and we can continue this conversation over there.